index.js

```diff
@@ -32,15 +32,49 @@ PLAYERS.forEach(renderPlayer)
 
 
 /***** Deliverable 1 *****/
+const header = document.querySelector("h1#header")
 function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+  if (element.target.style.color === "red") {
+    element.target.style.color = "black"
   } else {
-    element.style.color = "red"
+    element.target.style.color = "red"
   }
 }
-
+header.addEventListener("click", toggleColor)
 
 /***** Deliverable 2 *****/
+const addPlayerForm = document.querySelector("#new-player-form")
+
+function addPlayerSubmit(event) {
+  event.preventDefault()
+  const playerForm = event.target
+  
+  const number = playerForm.number.value
+  const name = playerForm.name.value
+  const nickname = playerForm.nickname.value
+  const photo = playerForm.photo.value
+
+  playerObj = {
+    number: number,
+    name: name,
+    nickname: nickname,
+    photo: photo,
+    likes: 1000
+  }
+
+  renderPlayer(playerObj)
+  event.target.reset()
+}
+
+addPlayerForm.addEventListener("submit", addPlayerSubmit)
+
 
 /***** Deliverable 3 *****/
+playerContainer.addEventListener("click", function(event) {
+  if (event.target.matches("button.like-button")) {
+    const player = event.target.closest(".player")
+    const likes = parseInt(player.querySelector(".likes").textContent.split(" ")[0])
+    player.querySelector(".likes").textContent = `${likes + 1} likes`
+  }
```

Comment:
Since you're using the `.likes` span a couple times, it might help to save it to a variable (this is also slightly more efficient, since the browser doesn't have to traverse the DOM twice):

  if (event.target.matches("button.like-button")) {
    const player = event.target.closest(".player")
    const likesSpan = player.querySelector(".likes")
    const likes = parseInt(likesSpan.textContent.split(" ")[0])
    likesSpan.textContent = `${likes + 1} likes`
  }

Also for `parseInt`, I like your approach because it's quite explicit about what the code's doing; if you want a slightly shorter way though:`parseInt` will actually strip out trailing characters for you (as long as the string starts with a number), so you can also do this:

```js
    const likes = parseInt(likesSpan.textContent)
```

Reply:
ohh interesting bit about parseInt! Thanks for the feedback. Best,